source/libtomo/accel/CMakeLists.txt

```
@@ -1,62 +1,71 @@
-# target properties based on build
-
-file(GLOB accel_sources
-    ${CMAKE_CURRENT_LIST_DIR}/cxx/*.cc)
-
-set(ACCEL_LIBRARY_SOURCES ${accel_sources})
+set(HEADERS "${tomopy_SOURCE_DIR}/include/libtomo/accel.h"
+            "${tomopy_SOURCE_DIR}/include/libtomo/macros.h")
+
+tomopy_add_library(
+  tomo-accel SHARED
+  cxx/common.cc
+  cxx/mlem.cc
+  cxx/sirt.cc
+  common.hh
+  constants.hh
+  cuda.h
+  data.hh
+  macros.hh
+  profiler.hh
+  typedefs.hh
+  utils.hh
+  ${HEADERS})
+
+target_include_directories(
+  tomo-accel
+  PUBLIC $<BUILD_INTERFACE:${tomopy_SOURCE_DIR}/include/libtomo>
+         $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include/libtomo>
```

```suggestion
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/libtomo>
```

For the installation interface, use a relative path to the install prefix. This allows the code to be relocatable and CMake is smart enough to detect it is a relative path and it will prefix the relative path with package root directory